docs/sources/tempo/api_docs/metrics-summary.md

```diff
@@ -30,6 +31,9 @@ overrides:
       processors: [..., 'local-blocks']
 ```
 
+In Grafana and Grafana Cloudl, the Metrics summary API is an [experimental feature](https://grafana.com/docs/release-life-cycle/) that is disabled by default.
```

Comment:
```suggestion
In Grafana and Grafana Cloud, the Metrics summary API is an [experimental feature](https://grafana.com/docs/release-life-cycle/) that is disabled by default.
```

Reply:
Now that its deprecated, we should no longer call it experimental but rather 'deprecated'.

Reply:
Makes me think we should have a "deprecated" entry on the 'release lifecycle' page: https://grafana.com/docs/release-life-cycle/

But that is a story for another day. :)

Reply:
We should. We do for the docs: https://grafana.com/docs/writers-toolkit/write/deprecate-remove/